services/web-server/src/monitor.js

```diff
@@ -4,4 +4,16 @@ const manager = new MonitorManager({
   serviceName: 'web-server',
 });
 
+manager.register({
+  name: 'createCredentials',
+  title: 'Credentials Created',
+  type: 'create-credentials',
+  version: 1,
+  level: 'info',
+  description: 'A client has been issued Taskcluster credentials',
+  fields: {
+    clientId: 'The client name.',
+  },
```

Comment:
Sorry for the double-drive-by here.  I should have looked harder last time.  I think this is issuing credentials to a *user* - so let's change this around to "A user has been issued Taskcluster credentials", with two fields:

```js
{
  clientId: 'The clientId of the issued credentials',
  userIdentity: 'The identity of the user to which the credentials were issued',
}
```

I think the latter can come from `user.identity`.

I think the last version had an `expires` property, too?  I can imagine the opsec team would want that in order to understand the timeframe of vulnerability from a compromised credential, for example.